js/jquery.mapael.js

```diff
@@ -413,16 +422,17 @@
                 panY: 0
             });
 
-            // init zoom button
-            $zoomIn = $("<div>").addClass(zoomOptions.zoomInCssClass).html("+");
-            $zoomOut = $("<div>").addClass(zoomOptions.zoomOutCssClass).html("&#x2212;");
-            self.$map.append($zoomIn).append($zoomOut);
-
-            $zoomIn.on("click." + pluginName, function () {
-                self.$container.trigger("zoom." + pluginName, {"level": self.zoomData.zoomLevel + 1});
-            });
-            $zoomOut.on("click." + pluginName, function () {
-                self.$container.trigger("zoom." + pluginName, {"level": self.zoomData.zoomLevel - 1});
+            // init zoom buttons
+            $.each(zoomOptions.buttons, function(type, opt) {
+                var $button;
+                // Create div with classes, contents and title (for tooltip)
+                $button = $("<div>").addClass(opt.cssClass)
+                                    .html(opt.content)
+                                    .attr("title", opt.title);
+                // Assign click event
+                $button.on("click." + pluginName, fnZoomButtons[type]);
```

Comment:
I was wondering it too :) Maybe I'm wrong but I don't think it will be a common need to add custom zoom buttons. Moreover, the user will still can add custom buttons freely, without using mapael conf.